package.json

```diff
@@ -154,6 +154,7 @@
     "@typescript-eslint/parser": "^5.59.5",
     "@xascode/tscpaths": "0.1.4",
     "axios-mock-adapter": "^1.21.2",
+    "bitcoin-core-ts": "^3.0.3",
```

Comment:
is it still used?

Reply:
yes, in tests... this PR has been replaced by https://github.com/GaloyMoney/galoy/pull/2655 but I will create another to clean tests